exchanges/bitfinex/bitfinex_test.go

```diff
@@ -892,22 +792,20 @@ func TestSubmitOrder(t *testing.T) {
 	}
 	response, err := b.SubmitOrder(context.Background(), orderSubmission)
 
-	if areTestAPIKeysSet() && err != nil {
+	if sharedtestvalues.AreAPIkeysSet(b) && err != nil {
 		t.Errorf("Could not place order: %v", err)
 	}
-	if areTestAPIKeysSet() && response.Status != order.New {
+	if sharedtestvalues.AreAPIkeysSet(b) && response.Status != order.New {
```

Comment:
Naturally, the very first test I experiment with caused a panic 😆 
If err != nil above, call t.Fatal there, because if there's an error, we can't check `response.Status`